app/config.py

```diff
@@ -15,7 +15,7 @@ class BaseConfig(object):
     LOG_BACKUPS = 0
     SECRET_KEY = 'houdini'
     # use sqlite by default
-    SQLALCHEMY_DATABASE_URI = os.getenv('DATABASE_URL', 'sqlite:///app.db')
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///app.db'
```

Comment:
I think you can leave this as it was - it gives a bit more flexibility to what db can be used. Simply leaving the `DATABASE_URL` env variable unset should default to the sqlite database.